spec/features/state_file/complete_intake_spec.rb

```diff
@@ -85,8 +86,7 @@
       expect(page).to have_text "File your Arizona state taxes for free"
       click_on "Get Started", id: "firstCta"
 
-      expect(page).to have_text "Next, set up your account with a quick code"
-      click_on "Text me a code"
+      step_through_initial_authentication(contact_preference: :text_message)
```

Comment:
I made a feature spec helper so that I didn't have to keep copying and pasting between the AZ & NY tests